README.md

```diff
@@ -1,25 +1,48 @@
 # Neovim Monokai
+
 Monokai Theme for Neovim with tree-sitter support
 
 <img width="1792" alt="monokai nvim" src="https://user-images.githubusercontent.com/25164326/121434903-0562f180-c94c-11eb-952d-df702d92fe25.png">
 
+Monokai Pro colors are also included.
+
 ### Prerequisites:
+
 Neovim Nightly (0.5) is required for monokai.nvim to work.
 
 ### Installation:
+
 #### Via Plug:
-```
+
+```viml
 Plug 'tanvirtin/monokai.nvim'
 ```
 
 #### Via Packer:
-```
+
+```lua
 use 'tanvirtin/monokai.nvim'
 ```
 
 ### Usage:
-```
+
+In VimL:
+
+```viml
 syntax on
-colorscheme monokai
 set termguicolors
+lua require('monokai').setup()
```

Comment:
If we add the line suggested in monokai.vim, we can remove this line from the README.md :)

Reply:
<img width="1792" alt="monokai_pro" src="https://user-images.githubusercontent.com/25164326/121989496-3f782d00-cd6a-11eb-9d60-a9fd5e16a87b.png">

We can also add the screenshot above as a sneak preview for this new feature.

Reply:
New method documented in README, see 4c203fc